melodic/distribution.yaml

```diff
@@ -3567,6 +3567,12 @@ repositories:
       url: https://github.com/ros-drivers/four_wheel_steering_msgs.git
       version: master
     status: maintained
+  foxglove_bridge:
+    release:
```

Comment:
I would recommend adding source and doc entries as well

Reply:
https://github.com/ros/rosdistro/pull/35383/commits/cfaece4afc3436904001616a802f678fbab61252